src/www/system_advanced_misc.php

```diff
@@ -397,6 +402,28 @@ function thermal_modules()
                 </td>
               </tr>
               <tr>
+                <td><i class="fa fa-info-circle text-muted"></i>  <?=gettext('On Battery Normal Mode') ?></td>
```

Comment:
`<td><a id="help_for_powerd_normal_mode" href="#" class="showhelp"><i class="fa fa-info-circle text-circle"></i></a>  <?=gettext('On Normal Power Mode'); ?></td>`

would this work?